src/main/java/com/mcmoddev/mmdbot/database/DatabaseManager.java

```diff
@@ -0,0 +1,24 @@
+package com.mcmoddev.mmdbot.database;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+/**
+ * @author Poke
+ */
+public class DatabaseManager {
+
+    public static Connection getConnection() {
```

Comment:
Could hold a database connection instance in the DatabaseManager instead of always creating a new one.

Reply:
What if we need multiple?

Reply:
The only case where you would want to do that is if you wanted to connect to an other _different_ database which I don't see happening. 

If, for some reason, it becomes needed, you can add an other variable/method to get a connection to the other database.

But again, I don't see this happening.

Reply:
I'd say that since there isn't yet a need for using multiple databases, we can rely on just one connection to one database for the lifetime of the bot.